sessions/how-to-code-review/pages/index.js

```diff
@@ -0,0 +1,106 @@
+import styled from "styled-components";
+
+import { Button } from "@/components/Button";
+
+import React, { useState } from "react";
+
+import myComponent from "@/components/myComponent1/myComponent.js";
+
+const kosmetikProdukte = [
```

Comment:
this could also be singular: 
```suggestion
const kosmetikProdukt = [
```

Reply:
Absolutely, yes